components/bl-transferList-component/src/list/list-item.js

```diff
@@ -0,0 +1,10 @@
+import { Checkbox } from './checkbox';
+
+export const ListItem = props => (
+  <label
+    htmlFor={ props.id }
+    className="list-item">
+    <Checkbox { ...props } />
```

Comment:
passing a spread is not a good practice, since in while it's quite tough to understand what exactly goes into the component 

for instance, you start passing a new property into the `ListItem` component and you. will need to omit the value from the Checkbox props